packages/operation/README.md

```diff
@@ -0,0 +1,257 @@
+yarn run build
+```
+
+## Airnode Development
+
+See below for more details
+
+### tl;dr
+
+```sh
+# Start Eth node (separate terminal)
+yarn run dev:eth-node
+
+# Deploy Airnode with API providers, templates, requesters etc. This creates a "deployment" file in a deployments/ folder.
+yarn run dev:eth-deploy
+
+# Make requests for Airnode to action
+yarn run dev:eth-requests
+
+# Airnode can then be invoked to process the requests. You must have a valid node development setup in place. You can find instructions in the node package README.
+yarn run dev:invoke
+```
+
+### Ethereum Development Node
+
+Start an Ethereum development node by running:
+
+```sh
+yarn run dev:eth-node
+```
+
+By default, this node listens on `http://127.0.0.1:8545/`. This is important as you will need to use this value in your `config.json` when running the Airnode node.
+
+This development node uses [Hardhat](https://hardhat.org/) behind the scenes. It creates no contracts by itself and only pre-funds a (configurable) number of accounts/addresses.
+
+### Deploying Airnode
+
+After starting an Ethereum development node, you can deploy the Airnode contracts to it by running:
+
+```sh
+yarn run dev:eth-deploy
+```
+
+Along with simply deploying the Airnode contracts, the above command will also create API providers onchain, deploy client contracts, create request templates and authorizers and several other things. See [Configuration](#Configuration) below for more information on customizing this behaviour.
+
+Running this command will build and save a "deployment" file in a `./deployments` folder. This file contains the addresses for the relevant accounts and contracts that are created. This is necessary as subsequent scripts do not have context of what these addresses and contracts are. You do not need to edit this file yourself.
+
+It is important to note that the Ethereum development node uses the same master mnemonic which means that the contracts will be deployed to the same addresses after restarting the node.
+
+### Making Requests
+
+Now that the contracts have been deployed and initial data setup, you can create some requests by running:
+
+```sh
+yarn run dev:eth-requests
+```
+
+A deployment file (`evm-dev.json`) must be present in the deployments folder before running this script.
+
+Airnode can now be invoked which will cause these requests to be actioned.
+
+## Configuration
+
+### Configuring deployment
+
+Deployment can be configured by adjusting the `config/eth-dev-config.json` file. This file has the following top level structure:
+
+```json
+{
+  "apiProviders": { ... },
+  "authorizers": { ... },
+  "clients": { ... },
+  "requesters": [],
+  "requests": []
+}
+```
+
+### 1. apiProviders
+
+`apiProviders` must have a unique name as the key.
+
+`mnemonic` - must be a unique 12 or 24 list of dictionary words. You can generate a mnemonic [here](https://iancoleman.io/bip39/). **DO NOT SEND REAL FUNDS TO A WALLET LINKED TO A TEST MNEMONIC**
```

Comment:
You can add here that the master wallet will be derived from the mnemonic, and master wallet address will also be the provider admin. I was a bit confused about not being able to find the provider admin address anywhere.